test/e2e/e2e.go

```diff
@@ -368,6 +409,7 @@ func LabelNamespace(clients *test.Clients, logf logging.FormatLogger, labels map
 	return err
 }
 
+// NamespaceExists creates a new namespace if it does not exist
 func NamespaceExists(t *testing.T, clients *test.Clients, logf logging.FormatLogger) (string, func()) {
```

Comment:
May be a better name for this function is CreateNamespaceIfNeeded or MakeSureNamespaceExists

Reply:
Thanks! Changed it to `CreateNamespaceIfNeeded`.